BasicSteps/SweepParameterStep.cs

```diff
@@ -273,6 +274,10 @@ public override void Run()
                     }
                     catch (TargetInvocationException ex)
                     {
+                        if (ex.InnerException is ArgumentException arEx)
```

Comment:
What about this? - its not always ArgumentException.
```suggestion
                        if (ex.InnerException is Exception arEx)
```